docker-compose.yml

```diff
@@ -11,6 +11,13 @@ services:
     environment:
       - NEO4J_AUTH=${NEO4J_USERNAME}/${NEO4J_PASSWORD}
       - NEO4JLABS_PLUGINS=["apoc"] 
```

Comment:
How about [referencing the `.env` file directly](https://docs.docker.com/compose/compose-file/#env_file)? There could potentially be distinct `local.env` and `prod.env`.

Reply:
The docker and docker-compose terminology for this is actually quite confusing. The variables coming from the `env_file` option are only for run-time so they would not be available in either docker-compose file or Dockerfile. Instead you have to use the `--env-file` command line option for them to be visible in the docker-compose file.

We have planned using different env files in MA-851.